app/src/main/kotlin/com/wire/android/notification/NotificationChannelsManager.kt

```diff
@@ -114,6 +121,22 @@ class NotificationChannelsManager @Inject constructor(
         notificationManagerCompat.createNotificationChannel(notificationChannel)
     }
 
+    private fun createPingNotificationChannel(userId: UserId, channelGroupId: String) {
+        val audioAttributes = AudioAttributes.Builder()
+            .setContentType(AudioAttributes.CONTENT_TYPE_SONIFICATION)
+            .setUsage(AudioAttributes.USAGE_NOTIFICATION)
+            .build()
+
+        val notificationChannel = NotificationChannelCompat
+            .Builder(NotificationConstants.getPingsChannelId(userId), NotificationManagerCompat.IMPORTANCE_HIGH)
+            .setName(NotificationConstants.PING_CHANNEL_NAME)
+            .setGroup(channelGroupId)
+            .setSound(knockSoundUri, audioAttributes)
```

Comment:
So the sound of ping is played by the notification, right? What if we have the conversation screen open? Then we don't show notifications for that specific conversation - how can the user hear the ping then?

Reply:
I will ask about it and do improvements in next PR